semantic.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Div soup page</title>
+</head>
+
+<body>
+ <!-- ovo je semanticki napisan kod koji ce jos sadrazavat i komentare  -->
+ <!-- ovo je Header dio koji sadrzi i  logo, naziv i navigaciju  -->
+ <header>
+  <img src="html logo.png" alt="html logo" width="150px">
+  <h1>Ovo je Semantic html stranica</h1>
+  <nav>
+   <a href="">Home</a>
+   <a href="#prvi">About Semantic</a>
+   <a href="#drugi">About SEO</a>
+   <a href="./index.html">Div soup</a>
+  </nav>
+ </header>
+
+ <!-- Ovo je main dio stranice koji sadrzi članke i forme -->
+
+ <main>
+  <!-- ode sam koristio section ali mislio sam mozda i article , jos uvijek neznam sta je bolje  -->
+
+  <section id="prvi">
+   <h3>What is Semantic Html</h3>
+   <p>Semantically correct HTML helps search engines, screen readers, and other user devices determine the significance
+    and context of web content. In order to effectively represent the meaning of content, appropriate HTML elements are
+    chosen according to inherent meaning, rather than how they appears visually on a rendered web page.
+   </p>
+   <img src="div_vs_semantic.jpeg" alt="div vs semantic " width="400px">
+  </section>
```

Comment:
@borissimic Bolje bi bilo da si odabra `<article>` iz razloga jer je sadržaj ovog bloka: "naslov, tekst i slika", što zapravo sadrži većina nekakvim članaka i ovaj blok je samostalan za sebe. Ali da, i `<section>` je dobar, sad je to već pametovanje s moje strane, ali oba mogu slobodno proći.